src/regions/de/info/travel-plans/travel.md

```diff
@@ -1,10 +1,12 @@
 ## 🧳Informationen und Sicherheitshinweise für Reisende und Personen mit Reiseabsichten
 
-**Das Auswärtige Amt hat eine weltweite Reisewarnung ausgesprochen:** Vor nicht notwendigen, touristischen Reisen in das Ausland wird derzeit gewarnt, da weiterhin mit starken drastischen Einschränkungen im internationalen Luft- und Reiseverkehr, weltweiten Einreisebeschränkungen, Quarantänemaßnahmen und der Einschränkung des öffentlichen Lebens in vielen Ländern zu rechnen ist. Dies gilt bis auf weiteres fort, vorerst bis einschließlich 14. Juni 2020.
+**Das Auswärtige Amt hat eine weltweite Reisewarnung ausgesprochen:** Vor nicht notwendigen, touristischen Reisen in das Ausland wird derzeit gewarnt, da weiterhin mit starken  drastischen Einschränkungen im internationalen Luft- und Reiseverkehr, weltweiten Einreisebeschränkungen, Quarantänemaßnahmen und der Einschränkung des öffentlichen Lebens in vielen Ländern zu rechnen ist. Dies gilt bis auf weiteres fort.
 
-Noch im Ausland befindliche Touristen sollten zurückreisen, solange es noch Reisemöglichkeiten gibt. Weitere Informationen finden Betroffene [hier](https://www.auswaertiges-amt.de/de/ReiseUndSicherheit/reise-und-sicherheitshinweise/reisewarnungen/faq-reisewarnung).
+Die Bundesregierung hat am 3. Juni 2020 entschieden, dass die Reisewarnung für die Mitgliedsstaaten der Europäischen Union, für Schengen-assoziierte Staaten und für das Vereinigte Königreich ab dem 15. Juni aufgehoben und durch individuelle Reisehinweise ersetzt werden soll.
 
-Informationen zu den eingeführten vorübergehenden Grenzkontrollen an den Binnengrenzen erhalten Sie [hier](https://www.bmi.bund.de/SharedDocs/faqs/DE/themen/bevoelkerungsschutz/coronavirus/coronavirus-faqs.html#doc13738352bodyText3).
+Noch im Ausland befindliche Touristen sollten zurückreisen, solange es noch Reisemöglichkeiten gibt.
+
+Zu den eingeführten vorübergehenden Grenzkontrollen an den Binnengrenzen und den Reisebeschränkungen im außereuropäischen Luftverkehr informiert das [Bundesinnenministerium auf seiner Internetseite](https://www.bmi.bund.de/SharedDocs/faqs/DE/themen/bevoelkerungsschutz/coronavirus/coronavirus-faqs.html;jsessionid=634537140F1CDF62BD58179FA582A9B8.2_cid373#doc13738352bodyText4).
```

Comment:
You don't need `jsessionid` here:
```suggestion
Zu den eingeführten vorübergehenden Grenzkontrollen an den Binnengrenzen und den Reisebeschränkungen im außereuropäischen Luftverkehr informiert das [Bundesinnenministerium auf seiner Internetseite](https://www.bmi.bund.de/SharedDocs/faqs/DE/themen/bevoelkerungsschutz/coronavirus/coronavirus-faqs.html#doc13738352bodyText4).
```

Reply:
BTW, this link leads to the `Quarantäne` section.
Should it rather point to `Reisebeschränkungen / Grenzkontrollen`?
```suggestion
Zu den eingeführten vorübergehenden Grenzkontrollen an den Binnengrenzen und den Reisebeschränkungen im außereuropäischen Luftverkehr informiert das [Bundesinnenministerium auf seiner Internetseite](https://www.bmi.bund.de/SharedDocs/faqs/DE/themen/bevoelkerungsschutz/coronavirus/coronavirus-faqs.html#doc13738352bodyText3).
```

Reply:
@YaraslauZhylko good catch. Yes it should rather point to Reisebeschränkungen / Grenzkontrollen.

Reply:
It still needs the `#doc13738352bodyText3` part in order to point to Reisebeschränkungen / Grenzkontrollen.